client/src/ui/CourseReviews.tsx

```diff
@@ -89,6 +89,11 @@ export class CourseReviews extends Component<Props, State> {
   reportReview = (review: ReviewType) => {
     Meteor.call('reportReview', review, (error: any, result: 1 | 0) => {
       if (!error && result === 1) {
+        let idx = 0;
+        while (idx < this.state.reviews.length && this.state.reviews[idx].key !== review._id) {
+          idx++;
+        }
+        this.setState({reviews: this.state.reviews.slice(0, idx).concat(this.state.reviews.slice(idx+1)) });
```

Comment:
I hate that this is one of the simplest ways to remove an element by index in JS but don't think there's another simpler way 👍